src/main/java/com/dreammaster/loginhandler/LoginHandler.java

```diff
@@ -46,6 +49,11 @@ public void onPlayerLogin(PlayerEvent.PlayerLoggedInEvent event) {
                                 .setChatHoverEvent(new HoverEvent(HoverEvent.Action.SHOW_TEXT, new ChatComponentText(YELLOW + CLICK_DISCORD)))
                                 .setChatClickEvent(new ClickEvent(ClickEvent.Action.OPEN_URL, Refstrings.DISCORD_LINK)))));
         event.player.addChatMessage(new ChatComponentText(GOLD.toString() + STRIKETHROUGH + "-----------------------------------------------------"));
+
+        MinecraftServer server = MinecraftServer.getServer();
+        if (server.isSinglePlayer() && !event.player.getGameProfile().getName().equals(server.getServerOwner())) {
+            event.player.addChatMessage(new ChatComponentText(OPEN_TO_LAN_WARNING));
```

Comment:
Since Open to LAN is nearly identical to singleplayer, I had to bend backwards a little to make sure this message is not sent to player running the server. I hope the check is robust enough.